### PR TITLE
Migrate beetmoverscript

### PR DESCRIPTION
**Pull request by garbas**

### CODE REVIEW

**Comment by MihaiTabara** [comment]
@garbas we need to green up the build-docker-image before merging this https://tools.taskcluster.net/groups/OUjqd8KYTC2GDP1UeANdkg/tasks/CYsXpMOgSNafYmn8ydyvSg/details

File: .taskcluster.yml [hunk 1]
```diff
@@ -10,13 +10,15 @@ tasks:
...
-      - ['treescript',    '37',             ''] # mozilla/releng-treescript'] # until we figure out how to install it
+      # [ <PROJECT NAME>,   <PYTHON VERSION>, <DOCKERHUB REPO>]
+      - ['client',          '36',             '']
+      - ['client',          '37',             '']
+      - ['iscript',         '36',             '']
+      - ['iscript',         '37',             '']
+      - ['treescript',      '36',             '']
+      - ['treescript',      '37',             ''] # mozilla/releng-treescript'] # until we figure out how to install it
+      - ['beetmoverscript', '36',             '']
+      - ['beetmoverscript', '37',             mozilla/releng-beetmoverscript']
```
Comment (escapewindow) [inline .taskcluster.yml]: It looks like we need to rename this. `mozilla/releng_beetmoverscript`?

`Invalid repository name (releng-beetmoverscript'), only [a-z0-9-_.] are allowed`

**Review by escapewindow — APPROVED** [review]
Thanks! Same comment as balrogscript - we should probably file issues about followups needed.

File: .taskcluster.yml (hunk 1 above)
Comment (escapewindow) [reply .taskcluster.yml]: Hm, maybe it's complaining about the unpaired `'`. We either need to add the missing `'` before `mozilla`, or we need to remove the trailing `'`.

Comment (garbas) [reply .taskcluster.yml]: ah, that was a typo. i missed one initial `'`

**Pull request merged**